tests/media_routing_loop_test.go

```diff
@@ -0,0 +1,133 @@
+package tests
```

Comment:
Please remove this file and the test within. Complement does not listen on `:8448` anymore so this won't work, and I'd want to have a standardised way of spinning up mini web-servers.

Reply:
Ditto https://github.com/matrix-org/complement/pull/279#discussion_r833165265